manifests/2/2BrightSparks/SyncBackSEX64/10.2.68.0/2BrightSparks.SyncBackSEX64.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 2BrightSparks.SyncBackSEX64

```

```suggestion
PackageIdentifier: 2BrightSparks.SyncBackSE
```
You'll have to change the `SyncBackSEX64` folder to `SyncBackSE` ...
... and update the file names as well.

You can do this by pressing the `.` (dot) key on your keyboard which will open up Visual Studio Online.